include/battle.h

```diff
@@ -732,6 +733,8 @@ struct BattleStruct
     bool8 transformZeroToHero[PARTY_SIZE][NUM_BATTLE_SIDES];
     u8 pledgeMove:1;
     bool8 isSkyBattle:1;
+    bool8 intrepidSwordBoost[PARTY_SIZE][NUM_BATTLE_SIDES];
+    bool8 dauntlessShieldBoost[PARTY_SIZE][NUM_BATTLE_SIDES];
```

Comment:
At first I let go the Zero to Hero one thinking it would've been a one-time thing, but seeing that there's more coming, let's have these as bitfields instead.

Reply:
Check `battleBondTransformed` as a reference.

Reply:
review applied